<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FIOModbus</class>
 <widget class="QWidget" name="FIOModbus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="DataButton" name="pushButton_in3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_relay1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_relay2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>打开设备</string>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_in4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_in1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_relay4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_server">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>串口号</string>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_relay3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_in2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>351</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新设备</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataButton</class>
   <extends>QPushButton</extends>
   <header>databutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
